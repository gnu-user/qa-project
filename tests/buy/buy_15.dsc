TITLE
    Ticket Purchase from Seller (Invalid Seller, does not exist)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the buy 
    transaction. When prompted, enter a title which exists. When prompted, 
    enter 4 tickets to purchase. When prompted, enter a seller which does not 
    exist. Verify that the seller is not accepted, and an error occurs.
DEPENDENCIES
    buy_14
