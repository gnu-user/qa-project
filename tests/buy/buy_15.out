Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the event title: 
Enter the number of tickets to purchase: 
Enter the seller's username: 
Invalid seller username, seller is not selling any tickets.
> 